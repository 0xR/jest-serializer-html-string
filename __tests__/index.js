/* eslint-env jest */

const unfmtHtml =
`<body>
			<div class="home active">
				<h1><img src="/foo.png" alt="Foo"/></h1><div>Hello</div>
				<p><span class="something">Hey</span>
				<img src="/foo.png" /></p>
			</div>
		</body>
`;

describe('jest-serializer-html-string', () => {
	it('should reformat a given html', () => {
		expect(unfmtHtml).toMatchSnapshot();
	});

	it('should not reformat something that is not an html string', () => {
		expect(2).toMatchSnapshot();
		expect(true).toMatchSnapshot();
		expect([]).toMatchSnapshot();
		expect({}).toMatchSnapshot();
		expect('Non html tags strings').toMatchSnapshot();
	});
});
