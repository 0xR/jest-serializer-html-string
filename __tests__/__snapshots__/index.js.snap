exports[`jest-serializer-html-string should not reformat something that is not an html string 1`] = `2`;

exports[`jest-serializer-html-string should not reformat something that is not an html string 2`] = `true`;

exports[`jest-serializer-html-string should not reformat something that is not an html string 3`] = `Array []`;

exports[`jest-serializer-html-string should not reformat something that is not an html string 4`] = `Object {}`;

exports[`jest-serializer-html-string should not reformat something that is not an html string 5`] = `"Non html tags strings"`;

exports[`jest-serializer-html-string should reformat a given html 1`] = `
<body>
  <div class="home active">
    <h1>
      <img src="/foo.png" alt="Foo" />
    </h1>
    <div>Hello</div>
    <p>
      <span class="something">Hey</span>
      <img src="/foo.png" />
    </p>
  </div>
</body>
`;
